src/network/http_server.rs

```diff
@@ -1726,7 +1736,7 @@ impl HubHttpService for HubHttpServiceImpl {
                     owner: format!("0x{}", hex::encode(&p.owner)),
                     signature: p.signature.clone(),
                     fid: p.fid,
-                    r#type: p.r#type,
+                    r#type: map_username_proof_enum_to_string(p.r#type).expect("Invalid type"),
```

Comment:
nit: can you use `p.type()` and avoid the error handling for invalid username proof type? this is generated by us so there's no reason for there to be an invalid type.